liquidator/liquidator.js

```diff
@@ -15,6 +15,20 @@ class Liquidator {
 
     // Instance of the expiring multiparty to perform on-chain liquidations.
     this.empContract = this.empClient.emp;
+
+    // Set default config settings. Caller can override these settings by passing in new
+    // values via the `config` input object.
+    const defaultConfig = {
+      crThreshold: this.web3.utils.toWei("0.98")
+      // `crThreshold`: Expressed as a percentage. If a position's CR is below the
+      // minimum CR allowed times `crThreshold`, then the bot will liquidate the position.
+      // This acts as a defensive buffer against sharp price movements delays in transactions getting mined.
+    };
+    Object.keys(defaultConfig).forEach(field => {
+      this[field] = config && config[field] ? config[field] : defaultConfig[field];
+      // TODO: Should we validate these fields? i.e. `priceThreshold < 1 && > -1`? If so, should I add a
+      // `validate: () => { return check_val }` arrow function to each defaultConfig field?
```

Comment:
+1 I think this is a cool idea.

Reply:
cool done, let me know what you think of this pattern